recipes/msys/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "msys" %}
+{% set version = "1.7.258" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - git_url: git://github.com/rmcgibbo/msys.git
+    git_rev: "cf53228"
+  - url: http://www.inchi-trust.org/download/105/INCHI-1-SRC.zip
+    sha1: e3872a46d58cb321a98f4fd4b93a989fb6920b9c
+    folder: external/inchi/INCHI-1-SRC
+  - url: https://sourceforge.net/projects/lpsolve/files/lpsolve/5.5.2.5/lp_solve_5.5.2.5_source.tar.gz
+    folder: external/lpsolve/lp_solve_5.5
+    sha1: 232ce8edb073c7c9b6d9100abf77038d989c45c8
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - scons
+    - python
+    - numpy
+    - boost>=1.67.0
```

Comment:
There should be a space after the `boost`, ie. `boost >=1.67.0`.  Also, are you sure you want boost and not boost-cpp?

Reply:
I'll give that a try

Reply:
I do want boost, rather than just boost-cpp (boost/python.hpp)